### ISSUE TITLE
Warning admonitions too bright in dark mode

### ISSUE THREAD

**Issue by jnns** [django/djangoproject.com#1313]
In my opinion, the background of the warning boxes is too bright in dark mode:

![grafik](https://user-images.githubusercontent.com/206281/218796954-7548a8a5-f2a2-441e-80c3-3a37aa9194f8.png)

I'm not sure if this intentional (after all, it's a warning). If this is the case, I'm happy with this issue getting closed. It's just so bright that I assume that it got overlooked when dark mode was implemented.

In general: thanks for implementing the dark mode to anyone involved!

**Comment by CuriousLearner** [comment]
Thanks for reporting this Jannis. You're welcome to submit a patch if you'd like, or let some new contributor submit a patch for this.

Thanks again!

**Comment by sabderemane** [comment]
> I'm not sure if this intentional (after all, it's a warning). If this is the case, I'm happy with this issue getting closed. It's just so bright that I assume that it got overlooked when dark mode was implemented.

It was deliberate to have a sufficient contrast in terms of accessibility, the initial color of the light mode has been modified to contrast sufficiently with the background.

**Comment by RyanWalker277** [comment]
Hey @sabderemane @CuriousLearner! Let me know if this needs a patch. I would be willing to work upon one.

**Comment by CuriousLearner** [comment]
If we do consider this change, we'll also have to change the boundary of warning messages and `.admonition-title::before` for the warning symbol to be of a darker shade. Anyways, this would need a little more input to decide if this should be worked upon.

**Comment by inbuilt-aura** [comment]
hey @jnns ,can you give me permission to work on this patch if you want?

**Comment by jnns** [comment]
You're welcome to work on this as far as I am concerned. But please discuss if and how changes should be made first.

**Comment by inbuilt-aura** [comment]
Sorry for the late reply,yeah sure let's discuss on the issue first and
what changes should we made.Let me just go through and i will report you
back asap!

**Comment by sabderemane** [comment]
> I think it can be tweaked to #93d7b7 or #c9f0dd to be more suitable with Dark mode, albeit it needs more input from @django/accessibility team.

I'm part of the team ✋🏽, but I think it can be confusing with the color change depending on the light theme for some people, IMO it's better to keep the same color until we change the color theme or redesign the project. Nevertheless, I can suggest another change that combines both, avoid too bright and colors I think.

<img width="994" alt="Capture d’écran 2023-02-27 à 14 06 36" src="https://user-images.githubusercontent.com/17890338/221572590-93a8121a-6dd2-47b3-a55e-09b9efe289ed.png">

I've done that quickly but colors of the border and other elements can be adjusted to have enough contrast.

**Comment by knyghty** [comment]
@sabderemane your version looks quite nice to me. Only concern I have is how visible it is that it's a warning but I think it's pretty good.

**Comment by jnns** [comment]
I'm not sure if my red-green-deficiency plays a role in this, but for me the warning in @sabderemane's version isn't as distinguished from the info box above as I want it to be.

**Comment by knyghty** [comment]
Hmm okay, I suppose someone should run through this with vision deficiency emulation in that case...

**Comment by Batman0003333** [comment]
I want to work on this issue pls assign me this issue.

**Comment by CuriousLearner** [comment]
Hi @knyghty 
 @sabderemane's version looks good, though I think the warning does not appear to be as distinctive as it is currently.

I agree with @thibaudcolas 's comment above on revisiting colors site-wide.

Turning the opacity of the icon to 1 certainly would help, and I was wondering if it should also have a background-color to be more distinctive though.

**Comment by YashGodwal** [comment]
I want to work on this issue pls assign me this issue.

**Comment by thibaudcolas** [comment]
@YashGodwal this is already being worked on by someone else, please take a moment to read existing conversation before claiming tasks.

**Comment by sabderemane** [comment]
For followers of the topic, here is the PR : https://github.com/django/djangoproject.com/pull/1360
